### ISSUE TITLE
Missing argument: learning_rate

### ISSUE THREAD

**Issue by nik1806** [JdeRobot/DeepLearningStudio#55]
The PilotNet training script in tensorflow needs a learning rate parameter as follow:
https://github.com/JdeRobot/DeepLearningStudio/blob/a174699c8b79d61a6207f5c8334990b4f7d3e0ba/Formula1-FollowLine/tensorflow/PilotNet/train.py#L39

But no such argument is defined:
https://github.com/JdeRobot/DeepLearningStudio/blob/a174699c8b79d61a6207f5c8334990b4f7d3e0ba/Formula1-FollowLine/tensorflow/PilotNet/train.py#L19-L25
